index.html

```diff
@@ -84,197 +46,274 @@ <h4>Zero-ops OpenStack on just about any Linux box.</h4>
+              <button class="p-code-copyable__action">Copy to clipboard</button>
+            </div>
+          </li>
+          <li class="p-stepped-list__item">
+            <h3 class="p-stepped-list__title">Launch a VM</h3>
 
-          <p>
-            That’s it! Your OpenStack is ready.
-          </p>
-        </div>
+            <div class="p-code-copyable u-stepped-list-margin">
+              <input class="p-code-copyable__input" value="microstack.launch cirros --name test" readonly="readonly">
+              <button class="p-code-copyable__action">Copy to clipboard</button>
+            </div>
+            <p>Your VM is up and running now. You should see a message like:</p>
+            <p>Access your server with <code class="highlighter-rouge">ssh -i /home/ubuntu/.ssh/id_microstack cirros@10.20.20.3</code></p>
+          </li>
+        </ol>
+        <p>
+          That’s it! Your OpenStack is ready.
+        </p>
       </div>
-      <div class="u-fixed-width">
-        <hr class="p-separator" />
+      <div class="col-6 js-tab__content" id="tab-two__content" role="tabpanel" aria-labelledby="tab-two">
+        <ol class="p-stepped-list u-no-margin--left">
+          <li class="p-stepped-list__item">
+            <h3 class="p-stepped-list__title">Install Multipass for Windows</h3>
+
+            <p class="p-stepped-list__content u-stepped-list-margin">
+              <a href="https://github.com/CanonicalLtd/multipass/releases" class="p-button--positive js-download"
+                data-os="win-win">
+                <span class="p-link--external">Download Multipass for Windows</span>
+              </a>
+            </p>
+          </li>
+          <li class="p-stepped-list__item">
+            <h3 class="p-stepped-list__title">Start a multipass VM</h3>
+
+            <div class="p-code-copyable u-stepped-list-margin">
+              <input class="p-code-copyable__input" value="multipass launch --name microstack-vm --mem 8G --disk 40G" readonly="readonly">
+              <button class="p-code-copyable__action">Copy to clipboard</button>
+            </div>
+          </li>
+          <li class="p-stepped-list__item">
+            <h3 class="p-stepped-list__title">Install the MicroStack snap on that multipass VM</h3>
+
+            <div class="p-code-copyable u-stepped-list-margin">
+              <input class="p-code-copyable__input" value="multipass exec microstack-vm -- sudo snap install microstack --classic --edge"
+                readonly="readonly">
+              <button class="p-code-copyable__action">Copy to clipboard</button>
+            </div>
+          </li>
+          <li class="p-stepped-list__item">
+            <h3 class="p-stepped-list__title">Configure networks and OpenStack databases</h3>
+
+            <div class="p-code-copyable u-stepped-list-margin">
+              <input class="p-code-copyable__input" aria-label="sudo snap install microk8s --classic"
```

Comment:
```suggestion
              <input class="p-code-copyable__input"
```
This looks like a copy-paste error … wrong command, wrong snap name, and most importantly, not an appropriate use of `aria-label`.